opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/ApplicationRestService.java

```diff
@@ -58,8 +68,17 @@
 @Transactional
 public class ApplicationRestService extends AbstractDaoRestService<OnmsApplication,OnmsApplication,Integer,Integer> {
```

Comment:
There are events sent for creation and deletion of an application but not for changing an application, right? Or am I missing something?

Reply:
That is correct. The ApplicationRestService has no changeApplication method?

Reply:
@christianpape I checked again. No edit method (the default implementation doesn't support edit).

Reply:
OK